src/js/components/form-elements/FilterInput.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { getTranslate } from 'react-localize-redux';
+import { connect } from 'react-redux';
+
+import { translateWithDefaultMessage } from 'utils/Translate';
+
+const FilterInput = ({
+  itemFilter, onChangeFunc, clearFunc, translate,
+}) => (
+  <div className="d-flex mr-auto justify-content-center align-items-center">
+    <input
+      value={itemFilter}
+      onChange={onChangeFunc}
+      className="float-left btn btn-outline-secondary btn-xs filter-input mr-1 mb-1"
+      placeholder={translate('react.stockMovement.searchPlaceholder.label', 'Search...')}
+    />
+    {itemFilter &&
+      <i
+        role="button"
+        className="fa fa-times-circle"
+        style={{ color: 'grey', cursor: 'pointer' }}
+        onClick={() => clearFunc()}
+        onKeyPress={() => clearFunc()}
```

Comment:
Could you explain why you have both `onClick` and `onKeyPress` assigned a same `clearFunction` ?

Reply:
The answer is that the ESLint shouts if you don't add at least one key listener for a static element such as `<i>` or `<div>`, so that is why it was probably added back in the past.

> Visible, non-interactive elements with click handlers must have at least one keyboard listener.